pkg/operator/ceph/csi/csi.go

```diff
@@ -20,136 +20,83 @@ import (
 	"strconv"
 
 	"github.com/pkg/errors"
-	"github.com/rook/rook/pkg/clusterd"
-	controllerutil "github.com/rook/rook/pkg/operator/ceph/controller"
 	"github.com/rook/rook/pkg/operator/k8sutil"
 	"k8s.io/apimachinery/pkg/version"
-	"k8s.io/client-go/kubernetes"
 )
 
-func ValidateAndConfigureDrivers(context *clusterd.Context, namespace, rookImage, securityAccount string, serverVersion *version.Info, ownerInfo *k8sutil.OwnerInfo) {
-	csiLock.Lock()
```

Comment:
Since the CSI config does not happen in the goroutine, this lock seems unnecessary. @travisn FYI.